mantle/kola/tests/misc/boot-mirror.go

```diff
@@ -203,6 +205,17 @@ func detachPrimaryBlockDevice(c cluster.TestCluster, m platform.Machine) {
 		if err := m.(platform.QEMUMachine).RemovePrimaryBlockDevice(); err != nil {
 			c.Fatalf("failed to delete the first boot disk: %v", err)
 		}
+		// Check for the boot ID before initiating a reboot
```

Comment:
Maybe here something like:

```suggestion
		// Check if we can still SSH into the machine. We've noticed sometimes
		// that after removing the primary device, we lose connectivity.
```